Trackers/iou_pred_tracker/iou_pred.py

```diff
@@ -51,6 +52,7 @@ def add_track(self, id, bbox):
         """
         
         track = iou_pred_track(id, bbox , 1, 0)
+        track.history = []
```

Comment:
If we have to initialize the variables like this then what is the need for concept name constructors of the object class? We should define them directly right?

Reply:
Done